packages/sdk/src/index.ts

```diff
@@ -1,3 +1,6 @@
-export * from './dataProtector/types/index.js';
-export { IExecDataProtector } from './dataProtector/IExecDataProtector.js';
+export * from './lib/types/index.js';
 export { getWeb3Provider } from './utils/getWeb3Provider.js';
+
+export { DataProtector } from './lib/dataProtector/DataProtector.js';
+export { DataProtectorSharing } from './lib/dataProtectorSharing/DataProtectorSharing.js';
```

Comment:
Exposing these two "child" constructors means that we will allow the user to directly instantiate `DataProtector`, in case maybe they are not interested in `sharing` methods?

Sounds good.

Reply:
yeah it was my idea 🙂